backend-rust/src/indexer.rs

```diff
@@ -3507,7 +3512,7 @@ async fn process_cis2_token_event(
 }
 
 struct PreparedScheduledReleases {
-    account_address: Arc<String>,
+    canonical_address: Vec<u8>,
```

Comment:
It would be better to use the `CanonicalAddress` type everywhere instead of a `Vec<u8>`, since this can be stack allocated.
Then when using it in a query `.0.as_slice()`

Reply:
Good point